website/docs/r/schema_template_bd.html.markdown

```diff
@@ -45,13 +45,20 @@ resource "mso_schema_template_bd" "bridge_domain" {
 * `ipv6_unknown_multicast_flooding` - (Optional) IPv6 Unknown Multicast Flooding behavior. Allowed values are `flood` and `optimized_flooding`. Default to `flood`.
 * `multi_destination_flooding` - (Optional) Multi-destination flooding behavior. Allowed values are `flood_in_bd`, `drop` and `flood_in_encap`. Default to `flood_in_bd`.
 * `unknown_multicast_flooding` - (Optional) Unknown Multicast Flooding behavior. Allowed values are `flood` and `optimized_flooding`. Default to `flood`.
+* `dhcp_policies` - (Optional) Block to provide dhcp_policy configurations. 
+* `dhcp_policies.name` - (Optional) Dhcp_policy name. Required if you specify the dhcp_policy.
+* `dhcp_policies.version` - (Optional) Version of dhcp_policy. Required if you specify the dhcp_policy.
+* `dhcp_policies.dhcp_option_policy_name` - (Optional) Name of dhcp_option_policy. 
+* `dhcp_policies.dhcp_option_policy_version` - (Optional) Version of dhcp_option_policy.
```

Comment:
Please change the documentation format.
For your reference: https://github.com/CiscoDevNet/terraform-provider-aci/pull/857/files#diff-0505816fc7d3615c5e4a081b8c2cf3e731a4b18fdb4d76f12bcf6e67ac7d3de1R107-R111

Reply:
ok